src/routes/LevelPage.jsx

```diff
@@ -89,16 +89,16 @@ const LevelPage = ({ history }) => {
 
   const NSFWWarningSwal = () => {
     ReactSwal.fire({
-      title: 'Include NSFW Content',
+      title: 'NSFW Content Included',
       html: (
         <>
-          This level include{' '}
-          <strong>NSFW(sensational or bloody) Content</strong>.
+          This level includes {' '}
```

Comment:
`includes {' '}` = `includes  `(two spaces)
please replace it with `includes{' '}`

Reply:
fixed in commit https://github.com/ADOFAI-gg/ADOFAI-gg-Web/pull/30/commits/53c46d2a7482c01e88e19bd1689bb62e59690c5f